app/aws/s3.py

```diff
@@ -82,27 +82,24 @@ def incr_jobs_cache_misses():
         redis_store.set(JOBS_CACHE_MISSES, 1)
     else:
         redis_store.incr(JOBS_CACHE_MISSES)
-    hits = redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
-    misses = redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
-    current_app.logger.debug(f"JOBS CACHE MISS hits {hits} misses {misses}")
+    redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
+    redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
```

Comment:
Do we still need the `get` statements here if we're not doing anything with the value?  All that needs to happen is the `set`, right?